### ISSUE TITLE
Fix RobotInit() Code

### ISSUE THREAD

**Issue by roshanr10** [FRCteam4909/BionicFramework#46]
So, finally ran the framework on PTC... having some unforeseen issues. Fixed some bugs in #45 but there's still a few remaining, don't think shouldn't be that hard to fix, just need to set aside some time to debug.

Most of the errors originate from this method: https://github.com/FRCteam4909/BionicFramework/blob/master/src/org/team4909/preseason2018/core/Robot.java#L26

**Comment by roshanr10** [comment]
UDPATE: Did some further research and I think we're having null pointer exceptions arising from our OI vs. Drivetrain

**Comment by roshanr10** [comment]
UDPATE: Did some further research and I think we're probably having null pointer exceptions arising from our OI vs. Drivetrain. 

[Usage of OI in Drivetrain Initialization](https://github.com/FRCteam4909/BionicFramework/blob/master/src/org/team4909/preseason2018/core/Robot.java#L39)

[Usage of Drivetrain in OI Initialization](https://github.com/FRCteam4909/BionicFramework/blob/master/src/org/team4909/preseason2018/core/OI.java#L24)

Could mitigate this by having another method within OI for buttons such as `initButtons()`. So in `RobotInit()`, we could code like so
 
```
oi = new OI()
drivetrain = new Drivetrain(...)
oi.initButtons()
```

**Comment by roshanr10** [comment]
RESOLVED in #47
